test/replication-luatest/suite.ini

```diff
@@ -2,4 +2,4 @@
 core = luatest
 description = replication luatests
 is_parallel = True
-release_disabled = gh_5295_split_brain_test.lua gh_6036_qsync_order_test.lua gh_6842_qsync_applier_order_test.lua gh_6033_box_promote_demote_test.lua gh_7253_election_long_wal_write_test.lua gh_7086_box_issue_promote_assert_test.lua
+release_disabled = gh_5295_split_brain_test.lua gh_6036_qsync_order_test.lua gh_6842_qsync_applier_order_test.lua gh_6033_box_promote_demote_test.lua gh_7253_election_long_wal_write_test.lua gh_7086_box_issue_promote_assert_test.lua gh_7590_replicaset_assert_test.lua
```

Comment:
AFAIU, we should prefer `t.tarantool.skip_if_not_debug()` right in the test code instead of the `release_disabled` list which is a remnant of diff-tests. But that is not 100%.

Reply:
@ylobankov could you please comment?

Reply:
I think it is better to use luatest tools to skip such tests. When you run this test via luatest directly, it will be skipped as well.